locales/fr.schema.json

```diff
@@ -1277,7 +1277,7 @@
           "label": "Afficher la deuxième image en survol"
         },
         "add_image_padding": {
-          "label": "Ajouter une marge intérieure à l'image"
+          "label": "Ajouter un remplissage d'image"
```

Comment:
This sounds weird 🤔

`Ajouter de l'espace au pourtour de l'image` would be more accurate. Thoughts? cc. @ludoboludo

Reply:
Im not sure if we should be changing those manually though 🤔  but I agree that their translation doesn't convey what the setting is doing. What you mention makes more sense to me as well.